src/core/src/preprocess/preprocess_steps_impl.cpp

```diff
@@ -688,6 +706,17 @@ std::tuple<std::vector<Output<Node>>, bool> PreStepsList::cut_last_channel(const
 }
 
 //------------- Post processing ------
+void PostStepsList::add_clamp(double min_value, double max_value) {
+    std::string name = "clamp(min " + std::to_string(min_value) + ", max " + std::to_string(max_value) + ")";
+
+    m_actions.emplace_back(
```

Comment:
What if some other actions are added before and after clamp? May it have any side effects from/to other actions?
I mean does order of actions matter?